package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "5.0.16",
```

Comment:
The current version is actually 5.0.17. I don't know why GH isn't showing that here.

Ref: https://github.com/bcgov/namerequest/blob/feature-way-of-navigating/package.json

Reply:
Obsolete issue.